install-dependencies.sh

```diff
@@ -52,6 +52,9 @@ debian_base_packages=(
 
 fedora_packages=(
     clang
+    compiler-rt
+    libasan
+    libubsan
```

Comment:
these two dependencies should be installed by seastar's `install-dependencies.sh`.  why do we need to include them explicitly here?

Reply:
still don't get this one, though.